C-Plus-Plus/dp/number_of_binary_trees.cpp

```diff
@@ -0,0 +1,34 @@
+#include <bits/stdc++.h>
```

Comment:
Have a beginner description about the Algorithms and the Code Implementation in a Multi-Line Comment.